WatchGuard patches to sources for bridge-utils-1.0.6 as of
Mon May  9 13:47:43 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- bridge-utils-1.0.6/libbridge/libbridge.h.orig	2022-05-09 13:47:43.050589501 -0700
+++ bridge-utils-1.0.6/libbridge/libbridge.h	2022-05-09 13:47:43.102587390 -0700
@@ -20,6 +20,7 @@
 #define _LIBBRIDGE_H
 
 #include <net/if.h>
+#include <linux/in6.h>
 #include <linux/if_bridge.h>
 
 struct bridge_id
